### PR TITLE
library build & UT running on netstandard

### PR DESCRIPTION
**Pull request by ckolumbus**
please check and let me know if this helps!

### CODE REVIEW

**Comment by crmckenzie** [comment]
Sweet! I'll check this out in the next 2 days. Sorry I can't be more specific than that.

**Comment by crmckenzie** [comment]
And I didn't get to it like I wanted to. Holiday havoc and all. My apologies.

**Comment by ckolumbus** [comment]
Hey, no problem. Happy Holidays and I'm glad if i can be of help!

File: Source/FizzWare.NBuilder/Extensions/TypeExtensions.cs [hunk 1]
```diff
@@ -9,17 +9,17 @@ public static class TypeExtensions
     {
         public static Type GetTypeWithoutNullability(this Type t)
         {
-            return t.IsGenericType &&
-                   t.GetGenericTypeDefinition() == typeof(Nullable<>)
-                       ? t.GetGenericArguments().Single()
+            return ReflectionHelper.GetTypeInfo(t).IsGenericType &&
```
Comment (lorddev) [inline Source/FizzWare.NBuilder/Extensions/TypeExtensions.cs]: @ckolumbus I used a similar method for this, but I used an extension method.

File: Source/FizzWare.NBuilder/Generators/RandomGenerator.cs [hunk 2]
```diff
@@ -72,9 +75,11 @@ public virtual decimal Next(decimal min, decimal max)
 
             int integer = rnd.Next(iMin, iMax);
             int fraction = rnd.Next(0, 4000);
-
+#if DNCORE
+            var separator = CultureInfo.CurrentCulture.NumberFormat.CurrencyDecimalSeparator;
```
Comment (lorddev) [inline Source/FizzWare.NBuilder/Generators/RandomGenerator.cs]: CultureInfo.CurrentCulture is backwards-compatible, so I don't think we need compiler directives for this.

Comment (ckolumbus) [inline Source/FizzWare.NBuilder/Generators/RandomGenerator.cs]: This was my first try in porting :-) thanks for the review and the feedback. We're going to make more use of NBuilder in the near future again so I'll alos look in further improvements regarding the porting and also the shared source across .Net 4 & Core

**Comment by lorddev** [comment]
@ckolumbus I noticed there's a failed unit test "ShouldBeAbleToUseMultipleTheFirsts()"... it's been in the master branch for a long time. What do you think we should do about it?

**Comment by ckolumbus** [comment]
@lorddev I'll check on Tuesday, right now we're on Easter holiday

**Comment by crmckenzie** [comment]
This issue was resolved in Issue #55

**Pull request closed without merging**